docker_registry2.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(/^(test|spec|features)\//)
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.6'
+  spec.add_development_dependency 'bundler'
```

Comment:
Why remove the constraint? Did it get in the way?

Reply:
It did. Unless I wanted to create an entirely new ruby gemset/installation that was just for Bundler 1. Any modern ruby installation is going to default to Bundler 2. Looking at the gemspec, I didn't see a reason to restrict it.

Reply:
Fair enough. Truth is, I cannot remember why I restricted it back then. :-)